examples/006_filter_composite_data_example.py

```diff
@@ -78,7 +78,9 @@
 # Get the stress field.
 stress_operator = composite_model.core_model.results.stress()
 stress_operator.inputs.bool_rotate_to_global(False)
-stress_field = stress_operator.get_output(pin=0, output_type=dpf.types.fields_container)[0]
+stress_field_material_coord = stress_operator.get_output(
```

Comment:
Should we mention what happens if this option is not set? What is the default?

Reply:
Good point. The default is true. I will add a comment